docs/remote.go

```diff
@@ -57,8 +57,8 @@ const (
 	RemoteListUse   string = `list`
 	RemoteListShort string = `List all singularity remote endpoints that are configured`
 	RemoteListLong  string = `
-	The 'remote list' command lists all remote endpoints configured for use. If you 
-	have set a remote as a default, its name will be encompassed by brackets.`
+	The 'remote list' command lists all remote endpoints configured for use. If a remote
+	has been set as default, its name will be encompassed by brackets.`
```

Comment:
Maybe?

```suggestion
	is in use, its name will be encompassed by brackets.`
```

CC @ikaneshiro

Reply:
@WestleyK's suggestion feels like the best way to phrase it